src/lightspeed_evaluation/core/output/statistics.py

```diff
@@ -3,7 +3,34 @@
 import statistics
 from typing import Any
 
-from lightspeed_evaluation.core.models import EvaluationResult
+import numpy as np
+import pandas as pd
+
+from ..models import EvaluationResult
```

Comment:
Revert this line (in latest PR, we changed this to absolute imports)

```suggestion
from lightspeed_evaluation.core.models import EvaluationResult
```

Reply:
Resolved it in new PR